src/server/client.cpp

```diff
@@ -166,13 +178,27 @@ namespace aperf {
         f.close();
       };
 
+      auto save_string = [](std::string path, std::string output) {
```

Comment:
Consider changing ```output``` to be either pass-by-reference or a pointer (pass-by-reference is better), as output strings can become very large.

Reply:
Done :+1: